### DISCUSSION TITLE
`value_and_hessian`?

### DISCUSSION THREAD

**Discussion by joglekara** [jax-ml/jax#14146] [Ideas] [answered]
Hi all,

Is there an efficient way to get `value_and_hessian` like we can with `value_and_grad`?

Thanks!

**Comment by jakevdp** [comment, marked as answer]
Not directly; see #762 for related discussion. Fortunately, there's a good workaround in many cases: just compute the value and the hessian normally as separate statements, wrap the whole function with JIT, and rely on the compiler for deduplication of operations.
